File: spring-boot-java-web/src/main/java/com/example/webjsp/model/SearchDefault.java
package com.example.webjsp.model;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class SearchDefault {
  /**
   * 검색조건
   */
  String searchCondition = "";

  /**
   * 검색Keyword
   */
  String searchKeyword = "";

  /**
   * 검색사용여부
   */
  String searchUseYn = "";

  /**
   * 현재페이지
   */
  int pageIndex = 1;

  /**
   * 페이지갯수
   */
  int pageUnit = 10;

  /**
   * 페이지사이즈
   */
  int pageSize = 10;

  /**
   * firstIndex
   */
  int firstIndex = 1;

  /**
   * lastIndex
   */
  int lastIndex = 1;

  /**
   * recordCountPerPage
   */
  int recordCountPerPage = 10;
}

File: spring-boot-kotlin-web/src/main/kotlin/com/example/webjsp/model/SearchDefault.kt
package com.example.webjsp.model

open class SearchDefault(
    var searchCondition: String = "",

    /** 검색Keyword  */
    var searchKeyword: String = "",

    /** 검색사용여부  */
    var searchUseYn: String = "",

    /** 현재페이지  */
    var pageIndex: Int = 1,

    /** 페이지갯수  */
    var pageUnit: Int = 10,

    /** 페이지사이즈  */
    var pageSize: Int = 10,

    /** firstIndex  */
    var firstIndex: Int = 1,

    /** lastIndex  */
    var lastIndex: Int = 1,

    /** recordCountPerPage  */
    var recordCountPerPage: Int = 10
)
